fix(Breadcrumb): variant multiple no longer same as default (#3345)
dnbexperience/eufemia: packages/dnb-design-system-portal/src/docs/uilib/components/breadcrumb/demos.mdx

---

import {
  BreadcrumbDefault,
  BreadcrumbChildren,
  BreadcrumbSingle,
  BreadcrumbCollapse,
  BreadcrumbMultiple,
} from 'Docs/uilib/components/breadcrumb/Examples'

## Demos

### Breadcrumb

> To ensure the correct use of the Breadcrumb, we recommend passing down pages as a variable to `data`. If you have other specific cases, check out how to customize with [children in Breadcrumb](/uilib/components/breadcrumb/#multiple-breadcrumb-with-children).

Reviewer: ```suggestion
To ensure the correct use of the Breadcrumb, we recommend passing down pages as a variable to `data`. If you have other specific cases, check out how to customize with [children in Breadcrumb](/uilib/components/breadcrumb/#never-collapse-variantmultiple).
```

The anchor needs to en aligned to the new title of the examle.

<BreadcrumbDefault />

- The first item, `Home`, gets assigned a home icon and an appropriate text label based on the current locale.
- The last item in pages will be static text, corresponding to the current page.
- Breadcumb will collapse on small screens

### Pages as child components

For customizing the Breadcrumb to fit your needs, you can add each page as a child component.

<BreadcrumbChildren />

### Single Breadcrumb

> When you only want a single button for `back`

Reviewer: ```suggestion
When you only want a single button for `back`.
```

<BreadcrumbSingle />

> This can also be forced using the `variant="single` prop.

Reviewer: ```suggestion
This can also be forced using the `variant="single"` prop.
```

> ### Always be collapsed (`variant="collapse`)

Reviewer: ```suggestion
### Always be collapsed (`variant="collapse"`)
```

Expands when user clicks

<BreadcrumbCollapse />

> ### Never collapse (`variant="multiple`)

Reviewer: ```suggestion
### Never collapse (`variant="multiple"`)
```
